ActiveLabeler-main/ActiveLabeler.py

```diff
@@ -0,0 +1,181 @@
+import random
+from tqdm import tqdm
+import numpy as np
+import torch
+from torch.utils.data import DataLoader
+from torchvision import transforms
+from torchvision.datasets import ImageFolder
+
+class ActiveLabeler:
+    """Active labeler code which is used with a SSL model to get curated dataset from unlabeled data"""
+
+    def __init__(self, embeddings, images_path,image_size,batch_size,seed):
+        """Initialise with embeddings and respected image paths"""
+        self.embeddings = embeddings
+        self.images_path = images_path
+        self.image_size = image_size
+        self.batch_size = batch_size
+        self.probabilities = []
+        np.random.seed(seed)
+        random.seed(seed)
+
+    def strategy(self, name, query, N, strategy_params):
```

Comment:
Upper case is usually used for constants. Please consider renaming this to `image_count`.